pkg/aspect/listsManager.go

```diff
@@ -67,7 +68,7 @@ func (listsManager) DefaultConfig() config.AspectParams {
 	}
 }
 
-func (listsManager) ValidateConfig(c config.AspectParams) (ce *adapter.ConfigErrors) {
+func (listsManager) ValidateConfig(c config.AspectParams, _ descriptor.Finder) (ce *adapter.ConfigErrors) {
```

Comment:
If any parameter is named all parameters must be named; since `c` is used in the body we have to have some name for the `Finder` param too.